Makefile

```diff
@@ -0,0 +1,44 @@
+SKILL_FOLDER := skills
+SAMPLE_SKILL_FOLDER := sample-skill
+
+build:
```

Comment:
this should not be called `build` but `setup` instead as this process only needs to be made once. Build is a process that compiles the source code in the executable bundle. If this is called build it would set the wrong expectation

Reply:
Done